### ISSUE TITLE
visu app - no positive response to user after linking already used device

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.matter#253]
matter adapter 0.3.2
visu app 1.3.1

- Device "Door" is provided by matter adapter and linked with google home
- Get in Google Home app the Kopplungcode for this connected device
- Run iobroker matter controller in virtual machine
- open iobroker VISU app -> Matter -> Answer question with "Ja es wird bereits verwendet"
- insert Kopplungscode of device in VISU app
- matter device is visible in matter controller in ioBroker 🟢 
- in visu app the screen stays there where you insert the code, there is no feedback to the user that it was successfull -> not ok 🔴 

![grafik](https://github.com/user-attachments/assets/c5331c3f-93ab-4597-9205-2cb33fcbe7c9)

[log.txt](https://github.com/user-attachments/files/18233938/log.txt)

**Comment by foxriver76** [comment]
Android right?

**Comment by Feuer-sturm** [comment]
@foxriver76 yes, Android.

**Comment by foxriver76** [comment]
Hm @Apollon77 visu App sends `controllerCommissionDevice` and checks in response for property `result === 'Ok'`

@Feuer-sturm and screen stays the same? Normally a toast should be shown and if successful go automatically back to matter settings

**Comment by Feuer-sturm** [comment]
@foxriver76 Yes, screen stays like in my screenshot. No confirmation and no return to matter settings

**Comment by Apollon77** [comment]
Please try next version, but ideally also try a BLE commissioning if possible

**Comment by Apollon77** [comment]
Ok we decided to update the visu app

**Comment by Apollon77** [comment]
@Feuer-sturm There is a new app version 1.3.2 where it should work now.

**Comment by Feuer-sturm** [comment]
matter 0.3.7
visu-app 1.3.2

- in visu app the screen stays there where you insert the code, there is no feedback to the user that it was successfull -> ok 🟢 

@foxriver76 It took ~5s after confirmation until the message "Gerät erfolgreich hinzugefügt" is shown. Is it possible to show a "in progress" icon to show to the user that pairing is in progress?
